### DISCUSSION TITLE
unsupported network error

### DISCUSSION THREAD

**Discussion by HeyokaLove13** [hashlips-lab/nft-erc721-collection#243] [Q&A]
Hey Guys! 

Thank you for all of your help and support! I've been baby dev-ing my way through the process and it's been FUN and painful! Not usually my sort of thing, but I'm learning so SO much!!! Thank you! 

I finally got the contract out and the dApp up! I'm extremely excited about it! I just have one small (hopefully) problem ... every time I try to connect my wallet I get an error that says "unsupported network" ... has anyone experienced this; and is there anyway I can fix it? I deployed everything on Polygon. The dApp acknowledges the contract is there -it acknowledges the network is there- it just doesn't seem to care for me very much. I assume I have to make an adjustment in the dApp somewhere. I'm just not sure where to look ...

**Comment by developerat79** [comment]
I having the same issue but the thing is that deployed the contract on Rinkeby Eth network

**Reply by lamefe** [reply]
I now have the same issue with Rinkerby. It did work before.

**Comment by lamefe** [comment]
please check solved issues: 
you need to change in collectionConfig.js
const CollectionConfig: CollectionConfigInterface = {
  testnet: Networks.**ethereumTestnet**,

**Reply by johncliffalvarez** [reply]
This definitely solved the issue! Thanks @lamefe!
